src/Sylius/Bundle/AttributeBundle/DependencyInjection/SyliusAttributeExtension.php

```diff
@@ -40,6 +46,15 @@ public function load(array $config, ContainerBuilder $container)
                 $formTranslationDefinition = $container->getDefinition('sylius.form.type.'.$name.'_translation');
                 $formTranslationDefinition->addArgument($parameters['subject']);
             }
+
+            if (isset($parameters['factory'])) {
+                $factoryDefinition = $container->getDefinition('sylius.factory.'.$parameters['subject'].'_attribute');
+                $factoryDefinition->addArgument($container->getDefinition('sylius.registry.attribute_type'));
+            }
+
+            if (strpos($name, 'value')) {
```

Comment:
I guess he want to check if $name contains 'value'. But if so, check should be done with comparison to false.
`if (false === strpos($name, 'value'))`